pywwt/core.py

```diff
@@ -892,6 +951,26 @@ def available_hips_catalog_names(self):
         """
         return sorted(self._available_hips_catalog_names)
 
+    # Support for source and HiPS catalog selection
+
+    _most_recent_source = None
+
+    @property
+    def most_recent_source(self):
+        """
+        The most recent source selected in the viewer.
+        """
+        return self._most_recent_source
+
+    _selected_sources = []
+
+    @property
+    def selected_sources(self):
+        """
+        A list of the selected sources, as dictionaries.
```

Comment:
The docstrings here need to give the reader a more specific idea of how the return values are actually structured. In this particular case I think that it would work well to point the reader to the research-app-messages API docs:

https://docs.worldwidetelescope.org/webgl-reference/latest/apiref/research-app-messages/interfaces/selections.source.html

These docs could have a bit more detail themselves (e.g. specifying the units for angles) but they'll help someone get an idea of the general shape of the data.

Reply:
Good idea, I'll point the reader in that direction. The `research-app-message` docs do give the units (of radians) for the angles, but they could use more information about the `catalogData` and `zoomDeg` fields.